jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -256,7 +257,16 @@ public JibContainerBuilder createJibContainerBuilder(
         Path war = getWarArtifact();
         Path explodedWarPath = tempDirectoryProvider.newDirectory();
         ZipUtil.unzip(war, explodedWarPath);
-        return JavaContainerBuilderHelper.fromExplodedWar(javaContainerBuilder, explodedWarPath);
+        return JavaContainerBuilderHelper.fromExplodedWar(
+            javaContainerBuilder,
+            explodedWarPath,
+            session
+                .getProjects()
+                .stream()
+                .map(MavenProject::getArtifact)
+                .map(Artifact::getFile)
+                .map(File::getName)
+                .collect(Collectors.toSet()));
```

Comment:
Like what you did in Gradle, you can refactoring this out to a variable and reuse it below as well.